cfgov/teachers_digital_platform/jinja2/teachers_digital_platform/activity_page.html

```diff
@@ -29,38 +29,40 @@ <h2 class="u-mt45">What students will do</h2>
     {{ page.what_students_will_do|richtext }}
     <div class="o-well u-mt45">
         <h2>Download activity</h2>
-            <h3 class="h5">Teacher guide</h3>
+        <h3 class="h5">Teacher guide</h3>
+        <p>
+            <a class="a-link a-link__icon" target="_blank" rel="noopener noreferrer" href="{{ page.activity_file.url }}">
+                <span class="a-link_text">{{ page.activity_file.title | striptags | safe }}</span>
+                <span class="a-icon">{{ svg_icon('download') }}</span>
+            </a>
+        </p>
```

Comment:
The link icons and all related markup (like the `span` around the text) are automatically added by our Python link parsing middleware, and the `p` tag is superfluous, so this can be simplified to just:

```suggestion
        <a href="{{ page.activity_file.url }}" target="_blank" rel="noopener noreferrer">
            {{- page.activity_file.title | striptags | safe -}}
        </a>
```

Please apply similar simplification to the following three links.

Reply:
I made these changes, although I needed to keep the `<p>` wrapper to provide enough spacing.

Reply:
Hm. I thought I tested it without it and removing the paragraphs made no difference. But maybe it did make a difference and I didn't think it looked any worse :)

Reply:
I think you forgot to make the link changes.

Reply:
(To commit and/or push them, I mean.)

Reply:
Gah! The link changes should be in now.